includes/data-stores/class-wc-admin-reports-orders-stats-data-store.php

```diff
@@ -75,6 +75,7 @@ class WC_Admin_Reports_Orders_Stats_Data_Store extends WC_Admin_Reports_Data_Sto
 	public static function init() {
 		add_action( 'woocommerce_refund_deleted', array( __CLASS__, 'sync_on_refund_delete' ), 10, 2 );
 		add_action( 'delete_post', array( __CLASS__, 'delete_order' ) );
+		add_action( 'save_post', array( __CLASS__, 'sync_order' ) );
```

Comment:
Is the existing order sync not sufficient here? https://github.com/woocommerce/wc-admin/blob/master/includes/class-wc-admin-reports-sync.php#L172

Reply:
Confirmed that this is unnecessary in my testing.

Reply:
@jeffstieler Thanks for catching that!  This hook is not necessary; just used to speed things up during testing and accidentally committed.